index.html

```diff
@@ -152,6 +152,17 @@ <h4>Pluggable raft log</h4>
                   Application can define the policies of how to write data and where it should be written easily.
               </p>
           </div>
+
+          <div class="col-md-6 col-sm-6 feature-item">
+              <span class="glyphicon glyphicon-dashboard" aria-hidden="true"></span>
+              <h4>Pluggable metrics</h4>
+              <p>Ratis provides a pluggable observation layer. By default, it uses dropwizard4. Users can provide their
```

Comment:
@SzyWilliam , thanks for working on this!  Let's also mention `ratis-metrics-dropwizard3`.
```
By default (ratis-metrics-default), it uses the shaded Dropwizard 4 provided by ratis-thirdparty.
Another implementation can be found in ratis-metrics-dropwizard3.
```

Reply:
Sure! Let me change the description.